osh.spec

```diff
@@ -187,6 +187,9 @@ rm -rf %{buildroot}%{python3_sitelib}/scripts
 
 # install example httpd config
 install -D {osh/hub,%{buildroot}%{_sysconfdir}/httpd/conf.d}/osh-hub-httpd.conf
+# configuration files should be kept under `/etc`
+mv %{buildroot}%{python3_sitelib}/osh/hub/settings_local.py %{buildroot}%{_sysconfdir}/osh/hub
+ln -s %{_sysconfdir}/osh/hub/settings_local.py %{buildroot}%{python3_sitelib}/osh/hub/settings_local.py
```

Comment:
When I execute `/usr/lib/python3.12/site-packages/osh/hub/manage.py` as `root` on my Fedora 39 VM, the Python interpreter creates `/usr/lib/python3.12/site-packages/osh/hub/__pycache__/settings_local.cpython-312.pyc`, which is not owned by any RPM package.  @xsuchy Is this expected/acceptable?

Reply:
We can modify `%post` to run `manage.py` as the `apache` user.  Still if someone runs the script(s) as `root` manually an unowned file will be created in `%{buildroot}%{python3_sitelib}`.  I take you answer as "this is acceptable packaing-wise".

Reply:
Right :)

Reply:
Fixed in de4295305b8aeccdb50b4100315ba08d9b571b61.